sdk/keyvault/Microsoft.Azure.Management.KeyVault/tests/KeyVaultTestBase.cs

```diff
@@ -54,7 +54,7 @@ public KeyVaultTestBase(MockContext context)
                 var graphClient = context.GetServiceClient<GraphRbacManagementClient>();
                 graphClient.TenantID = this.tenantId;
                 graphClient.BaseUri = testEnv.Endpoints.GraphUri;
-                this.objectId = graphClient.User.Get(testEnv.UserName).ObjectId;
+                this.objectId = testEnv.UserName;
```

Comment:
I'm not sure of the reasoning behind this change. I assume that the object id is being used to construct access policies. In this case the object id is not interchangeable with username.

Reply:
Glad you reviewed! In fact, I couldn't get it working, `graphClient.User.Get()` always threw an cloud exception without meaningful error message.
That's why I chose not to call graph, but pass in object ID directly.

Could you share if there are any special prerequisites required? Here are the environment variables I used:
```batch
set TEST_CSM_ORGID_AUTHENTICATION=SubscriptionId=***;ServicePrincipal=***;ServicePrincipalSecret=***;AADTenant=***;Environment=Prod;HttpRecorderMode=Record;UserId=***;
set AZURE_TEST_MODE=Record
```
Thanks